blockprod/src/rpc.rs

```diff
@@ -22,18 +22,18 @@ use common::{
 
 use serialization::{hex::HexDecode, hex::HexEncode};
 
-use crate::{detail::JobKey, BlockProductionError};
+use crate::{detail::job_manager::JobKey, BlockProductionError};
 use subsystem::subsystem::CallError;
 
 #[rpc::rpc(server, namespace = "blockprod")]
 trait BlockProductionRpc {
     /// Stop block production
     #[method(name = "stop_all")]
-    async fn stop_all(&self) -> rpc::Result<()>;
+    async fn stop_all(&self) -> rpc::Result<usize>;
 
     // Stop a specific job
     #[method(name = "stop_job")]
-    async fn stop_job(&self, job_id: JobKey) -> rpc::Result<()>;
+    async fn stop_job(&self, job_id: String) -> rpc::Result<bool>;
```

Comment:
Maybe also something to query if given job is running? Is that possible and useful?

Reply:
Sounds good, but I've held off until PoS is implemented, because that's the one that's going to drive the API for this possible call